common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst

```diff
@@ -14,7 +14,7 @@ Overview
 ========
 
 The MAVLink protocol defines a large number of
-`MAV_CMD <https://github.com/mavlink/mavlink/blob/master/message_definitions/v1.0/common.xml#L531>`__
```

Comment:
We should update the line number from 531 to 1008

Reply:
fixed